fvm/src/gas/price_list.rs

```diff
@@ -64,217 +64,72 @@ macro_rules! total_enum_map {
     };
 }
 
-/// Help create scaling costs in the format of `m*x+b` (conventional from the linear regression variables).
-macro_rules! scaling_mg {
-    ($m:literal * x + $b:literal) => {
-        ScalingCost::new(Gas::from_milligas($b), Gas::from_milligas($m))
-    };
-}
-
 lazy_static! {
-    static ref OH_SNAP_PRICES: PriceList = PriceList {
-        storage_gas_multiplier: 1300,
+    static ref HYGGE_PRICES: PriceList = PriceList {
+        // START (Copied from SKYR_PRICES)
 
-        on_chain_message_compute_base: Gas::new(38863),
-        on_chain_message_storage_base: Gas::new(36),
-        on_chain_message_storage_per_byte: Gas::new(1),
+        on_chain_message_compute: ScalingCost::fixed(Gas::new(38863)),
+        on_chain_message_storage: ScalingCost {
+            flat: Gas::new(36),
+            scale: Gas::new(1300),
+        },
 
-        on_chain_return_value_per_byte: Gas::new(1),
+        on_chain_return_compute: ScalingCost::zero(),
+        on_chain_return_storage: ScalingCost {
+            flat: Zero::zero(),
+            scale: Gas::new(1300),
+        },
 
+        // TODO(#1332)
         send_base: Gas::new(29233),
         send_transfer_funds: Gas::new(27500),
         send_transfer_only_premium: Gas::new(159672),
         send_invoke_method: Gas::new(-5377),
 
         create_actor_compute: Gas::new(1108454),
-        create_actor_storage: Gas::new(36 + 40),
-        delete_actor: Gas::new(-(36 + 40)),
+        create_actor_storage: Gas::new((36 + 40) * 1300),
+        delete_actor: Gas::new(-(36 + 40)*1300),
 
         sig_cost: total_enum_map!{
             SignatureType {
-                Secp256k1 => ScalingCost::new(Gas::new(1637292), Gas::zero()),
-                BLS       => ScalingCost::new(Gas::new(16598605), Gas::zero())
-            }
-        },
-
-        secp256k1_recover_cost: Gas::new(1637292), // TODO measure & revisit this value
```

Comment:
Should we dropping this TODO here?

Reply:
Yes. It's the same flat fee (the linear fee is due to hashing).